openbsd/Platform.c

```diff
@@ -392,7 +392,7 @@ void Platform_getBattery(double* percent, ACPresence* isOnAC) {
 
    *isOnAC = AC_ERROR;
    if (found) {
-      mib[3] = 9;
+      mib[3] = SENSOR_INDICATOR;
       mib[4] = 0;
```

Comment:
Note: `AC Power Status Indicator`

Reply:
I think I can document this in a better way…